### PR TITLE
(Re-)Implement BT and WiFi identification using Callsign

### PR DESCRIPTION
**Pull request by phl0**
This is a remake of https://github.com/bt-trx/firmware/pull/39 implementing all comments by @chris007de

### CODE REVIEW

File: src/bttrx_control.cpp [hunk 1]
```diff
@@ -255,6 +279,29 @@ string BTTRX_CONTROL::ParameterTypeToString(ParameterType parameter_type) {
...
+/**
+ * @brief Set Callsign
+ *
+ * @param callsign
+ * @return ResultType
+ */
+ResultType BTTRX_CONTROL::handleSetCallsign(string callsign) {
+  if (callsign.length() > CALLSIGN_LENGTH) {
+    serial_->dbg_println("Callsign exceed maximum length. Max. length is: " +
+                         CALLSIGN_LENGTH);
```
Comment (chris007de) [inline src/bttrx_control.cpp:291]: In this case, please `return kError;`

File: src/bttrx_wifi.cpp [hunk 2]
```diff
@@ -152,9 +152,14 @@ void BTTRX_WIFI::setup(BTTRX_CONTROL *control) {
   }
   bttrx_control_ = control;
 
+  Serial.println("");
   Serial.println("Configuring access point...");
 
-  WiFi.softAP(WIFI_SSID, WIFI_PASSWORD);
+  string ssid = BTTRX_WIFI::buildSSID(WIFI_SSID, bttrx_control_->getCallsign());
```
Comment (chris007de) [inline src/bttrx_wifi.cpp:158]: Please change name of `WIFI_SSID` to `WIFI_SSID_PREFIX`

File: src/bttrx_control.cpp (hunk 1 above)
Comment (phl0) [reply src/bttrx_control.cpp:291]: Fixed with https://github.com/bt-trx/firmware/pull/40/commits/5927cc77ca56e754ee0d6930ec983344373e2f63

File: src/bttrx_wifi.cpp (hunk 2 above)
Comment (phl0) [reply src/bttrx_wifi.cpp]: Fixed with https://github.com/bt-trx/firmware/pull/40/commits/5927cc77ca56e754ee0d6930ec983344373e2f63

**Pull request merged**